.drone.yml

```diff
@@ -3,7 +3,8 @@ name: default
 
 services:
   - name: docker
-    image: docker:dind
+    image: docker:19.03-dind
+    entrypoint: [dockerd]
```

Comment:
want to add 
```
command:
  - --log-level=error   
```
to suppress docker info logs?

Reply:
I see no harm in keeping them.

Reply:
Plus if we do that we have to specify the entire `command` argument.

Reply:
> I see no harm in keeping them.
 they provide no value, so it just takes up space

> Plus if we do that we have to specify the entire command argument.
shouldnt need to; we do this with talos to set the dns servers https://github.com/talos-systems/talos/blob/master/.drone.yml#L12

Reply:
> they provide no value, so it just takes up space

It provided value in figuring out that its generating TLS :)